frontend/src/expenditurePage/ExpenditurePage.js

```diff
@@ -0,0 +1,32 @@
+import React, {useContext} from 'react';
+import EventContext from "../contexts/EventContext";
+import {useHistory, useParams} from "react-router-dom";
+import Card from "@material-ui/core/Card";
+import {Button} from "@material-ui/core";
+
+export default function ExpenditurePage() {
+  const { events } = useContext(EventContext);
+  const { id, expenditureId } = useParams();
+  const event = events.find((event) => event.id === id);
+  const expenditure = event?.expenditures.find((exp) => exp.id === expenditureId)
+  const history = useHistory();
+  
+  console.log(expenditure);
+
+  return (
+    expenditure ? (
+      <>
+        <p>{expenditure.description}</p>
+        <p>{(expenditure.amount / 100).toFixed(2)} €</p>
+        <p>Bezahlt von: {expenditure.payer.firstName} {expenditure.payer.lastName.substring(0,1)}.</p>
```

Comment:
`{(expenditure.amount / 100).toFixed(2)} €` is repeated on line 25 and https://github.com/HennyHaelbich/Abschluss-Projekt-Reisekasse/blob/81c3dec2c9ad0ba5a1b95d3aca360a0f112db43b/frontend/src/eventPage/Expenditures.js#L28

same for 

`{expenditure.payer.firstName} {expenditure.payer.lastName.substring(0,1)}.` ion line 24 and https://github.com/HennyHaelbich/Abschluss-Projekt-Reisekasse/blob/81c3dec2c9ad0ba5a1b95d3aca360a0f112db43b/frontend/src/eventPage/Expenditures.js#L29

Maybe refactor to a helper function?